nix/set-git-rev/flake.nix

```diff
@@ -0,0 +1,22 @@
+{
+  inputs = {
```

Comment:
I wonder if there is no chance to reuse those things - they are not also used elsewhere?